### PR TITLE
Link std::process::Output to Command and Child

### PR DESCRIPTION
**Pull request by mistodon**
As per #29370

### CODE REVIEW

**Comment by carols10cents** [comment]
r? @steveklabnik 

Thanks for the PR! We’ll periodically check in on it to make sure that @steveklabnik or someone else from the team reviews it soon.

File: src/libstd/process.rs [hunk 1]
```diff
@@ -701,7 +701,14 @@ impl AsInnerMut<imp::Command> for Command {
     fn as_inner_mut(&mut self) -> &mut imp::Command { &mut self.inner }
 }
 
-/// The output of a finished process.
+/// The output of a finished process. This is returned in a Result by
```
Comment (steveklabnik) [inline src/libstd/process.rs]: Could you put some newlines between the first and second sentences here? The first sentence operates as a summary, so it's helpful to add space.

```rust
/// The output of a finished process.
///
/// This is returned in a Result...
```

**Review by steveklabnik — COMMENTED** [review]
Thank you! One small change, and afterwards, this is good to merge!

File: src/libstd/process.rs (hunk 1 above)
Comment (Pirh) [reply src/libstd/process.rs]: Done. I think that does look neater :)

**Pull request merged**